libs/ngx-dsfr/tag/src/lib/testing/tag.component.spec.ts

```diff
@@ -150,7 +130,6 @@ describe('DsfrTagComponent', () => {
     });
 
     it('should send a selected event when we select a tag', async () => {
-      testHost.testLabel = testLabel;
       testHost.testType = TagType.SELECTABLE;
       testHost.testSelected = testSelected;
       dsfrTagHarness = await harnessLoader.getHarness<DsfrTagHarness>(
```

Comment:
It enables the test to pass because of a side-effect of this action. But this action in itself is not necessary here because we already got the Harness in the `beforeEach`.

So we simply need to add the side-effect here so that we don't do anything "extra" here.

You need to replace this line with : `fixture.detectChanges();`